config/ie/installAll

```diff
@@ -30,9 +31,35 @@ def gafferRegistryVersion() :
 	if varsToFind:
 		raise Exception( "Could not find the gaffer version in the SConstruct file. Please review the parsing rules." )
 
-	return varsFound["gafferMilestoneVersion"] + "." + varsFound["gafferMajorVersion"] + ".0.0"
+	return ".".join( [varsFound[k] for k in versionVars] )
 
-gafferRegVersion = gafferRegistryVersion()
+
+def gafferRegistryVersion( gafferVersion ) :
+
+	# have to do local imports for some reason on the options file
+	import distutils
+	import IEEnv
+
+	gafferCompatibilityVersion = ".".join( gafferVersion.split( "." )[:2] )
+	looseGafferVersion = distutils.version.LooseVersion( gafferVersion )
+	registryVersions = list( IEEnv.registry["apps"]["gaffer"].keys() )
+	# only keep entries that have the expected elements
+	registryVersions = [x for x in registryVersions if len( x.split( "." ) ) == 4]
+	# only keep those that are of the same compatibility version
+	registryVersions = [x for x in registryVersions if x.startswith( gafferCompatibilityVersion )]
```

Comment:
This would accept `0.6` as a match for `0.60`. I'm not sure if that matters in practice, given the subsequent sorting...

Reply:
You are right. I should add a dot at the end of the `gafferCompatibilityVersion` to avoid that. Thanks.

Reply:
Fixed.